python/datasources/acs_health_insurance.py

```diff
@@ -126,490 +129,214 @@ def write_local_files_debug(self):
     # Returns:
     # FileDiff = If the data has changed by diffing the old run vs the new run.
     # (presumably to skip the write to bq step though not 100% sure as of writing this)
-    def upload_to_gcs(self, bucket):
+    def upload_to_gcs(self, bucket, **attrs):
```

Comment:
does this method need any test coverage?

Reply:
I don't think its worth it, its simple and would be hard to test

Reply:
cool, ya i think the same for the acs_population one i'll skip it